### PR TITLE
[release-4.11] Bug OCPBUGS-3295: Event Pull Status Notification GET Method not working as per o-Ran spec

### PR DESCRIPTION
**Pull request by aneeshkp**
fix get current state for events

### CODE REVIEW

File: plugins/ptp_operator/ptp_operator_plugin.go [hunk 1]
```diff
@@ -175,6 +160,73 @@ func Start(wg *sync.WaitGroup, configuration *common.SCConfiguration, fn func(e
...
+// getCurrentStatOverrideFn is called when current state is received by rest api
+func getCurrentStatOverrideFn() func(e v2.Event, d *channel.DataChan) error {
+	return func(e v2.Event, d *channel.DataChan) error {
+		if e.Source() != "" {
+			log.Infof("setting return address to %s", e.Source())
+			d.ReturnAddress = pointer.StringPtr(e.Source())
+		}
+		log.Infof("got status check call,send events for subscriber %s => %s", d.ClientID.String(), e.Source())
+		var eventType ptp.EventType
+		if !strings.Contains(e.Source(), string(ptp.PtpLockState)) &&
```
Comment (jzding) [inline plugins/ptp_operator/ptp_operator_plugin.go:172]: could just add a `else` statement at line 184

File: examples/consumer/deployment/namespace.yaml [hunk 2]
```diff
@@ -5,3 +5,7 @@ metadata:
   labels:
     name: cloud-events
     openshift.io/cluster-monitoring: "true"
+    security.openshift.io/scc.podSecurityLabelSync: "false"
+    pod-security.kubernetes.io/audit: "privileged"
+    pod-security.kubernetes.io/enforce: "privileged"
+    pod-security.kubernetes.io/warn: "privileged"
```
Comment (jzding) [inline examples/consumer/deployment/namespace.yaml:11]: what do these do?

File: plugins/ptp_operator/ptp_operator_plugin.go (hunk 1 above)
Comment (aneeshkp) [reply plugins/ptp_operator/ptp_operator_plugin.go:172]: can we turn off that  earl return linter error?

File: plugins/ptp_operator/ptp_operator_plugin.go [hunk 3]
```diff
@@ -175,6 +160,73 @@ func Start(wg *sync.WaitGroup, configuration *common.SCConfiguration, fn func(e
...
+			!strings.Contains(e.Source(), string(ptp.OsClockSyncState)) &&
+			!strings.Contains(e.Source(), string(ptp.OsClockSyncState)) {
```
Comment (jzding) [inline plugins/ptp_operator/ptp_operator_plugin.go:174]: I prefer to not disable and lint checks. If we keep this then need to fix this line. I believe it should check for `ptp.PtpClockClass` instead.

Comment (aneeshkp) [reply plugins/ptp_operator/ptp_operator_plugin.go]: good catch, Fixed that now

**Pull request merged**